### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by danielhickman** [crutchcorn/pinbadge-icon-theme#12]
## Pixel Perfection
Inner Circle: `172.521 px^2` at `(9.74 px, 9.74 px)`
![image](https://cloud.githubusercontent.com/assets/5341898/10267206/eaf968c2-6a3f-11e5-9b01-e445595eedf4.png)

## Layers Hierarchy
It might also be helpful to contributors (me and my automation included) if the layers hierarchy is uniform.
Things that might be helpful:

1. Naming first layer
2. Grouping edges
3. Tint always appearing first on the list before shade
4. Fix spelling error: "Outter" to "Outer"
5. Name Outer Circle Edge Group
6. Compound path for multi path glyphs

### From
```html
Layer 1
 |
 ---- Shade  | <path>
 ---- Tint | <path>
 ---- Glyph  | <path> or <group>
 ---- Inner Circle  | <circle>
 ---- Outter Circle | <group>
     |
     ---- <Group> | <group>
         |
         ---- Circle Tint | <path>
         ---- Circle Shade | <path>
     ---- Outter Circle <circle>
```

### To
```html
Icon
 |
 ---- Glyph Edges <group>
     |
     ---- Tint  | <path>
     ---- Shade | <path>
 ---- Glyph  | <path> or <compound-path>
 ---- Inner Circle  | <circle>
 ---- Outer Circle | <group>
     |
     ---- Circle Edges | <group>
         |
         ---- Circle Tint | <path>
         ---- Circle Shade | <path>
     ---- Outer Circle <circle>
```

## Guidelines
- Do colors have to be from the Material Palette? (If not, I'll work on an HSL based color formula for similar colors like #DACODE 500 to #DACODE 100)
- You should clarify that the colors you mentioned are from the Material Palette
- Should we really be using subpixel dimensions? `Whole circle is 45.5px x 45.5px` & `You should use 6.25px spaces between other objects`
- Add swatches for gradients or even better, graphic styles for them
- Centering. Do you eyeball shapes like Twitter, triangles, etc?

## Colors
I was thinking of making each icon a single color so someone like me can use a one tone desktop. I'd need to know which colors you'd think people'd like.

**I can make a few PRs adding all of this. And amazing work Corbin!**

**Comment by crutchcorn** [comment]
## Corbin Dicking Around
Okay 
1 - fuck you. 
2 - fuck YOU. 
3 - DON'T YOU FUCKING COME IN HERE AND TELL ME HOW TO SPELL OUTTTTTTTTTTER OKAY???

No but seriously - 
## Pixel Perfection
I did that for some reason but I cannot remember what. If you would like to modify all the AI files and SVGs for me that'd be greaaat. But if not then it's just gonna stay like that for a while. I'm lazy - I know.

## Layers Hierarchy
For the most part I do have a layers hierarchy. However, I can always improve it.
However, this doesn't work. 
```html
Icon
 |
 ---- Glyph Edges <group>
     |
     ---- Tint  | <path>
     ---- Shade | <path>
 ---- Glyph  | <path> or <compound-path>
 ---- Inner Circle  | <circle>
 ---- Outer Circle | <group>
     |
     ---- Circle Edges | <group>
         |
         ---- Circle Tint | <path>
         ---- Circle Shade | <path>
     ---- Outer Circle <circle>
```
Because the Glyph Shade needs to be BELOW the glyph at times. 
So rather, it will have to be
### To this:
```html
Icon
 |
 ---- Glyph Tint  | <path>
 ---- Glyph  | <path> or <compound-path>
 ---- Glyph Shade | <path>
 ---- Inner Circle  | <circle>
 ---- Outer Circle | <group>
     |
     ---- Circle Edges | <group>
         |
         ---- Circle Tint | <path>
         ---- Circle Shade | <path>
     ---- Outer Circle <circle>
```
## Guidelines
- I thought I put in the guidelines that you must use material colors, but I didn't. I'll add that now.
- Daniel, you know me - so fuck off with your 'you should mention they're fukin MD colors' /s
- Those are really just notes for me - you should pay little to no mind to them for now.
- I'd prefer them NOT to be swatches or anything of the sort - but rather hand picked. I still haven't written up all the guidelines that I am using for gradients either. EG: Being allowed to use Orange AND Deep Orange.
- Centering. I use circle centering for those objects.

## Colors
For that - please continue to discuss with me via hangouts about where to place that. I think I might want a new repo for it. What do you think - time for a Pinbadge org on GH? If not, then I'll allow you to make that your project based off of my work. This is all open source after all for a reason. I personally would like Gray, Blue Gray, Green. Blue (and of them, preferably light blue), and IDK what else.

** Thank you much, love **

**Comment by crutchcorn** [comment]
I've JUST finished editing the AI files to conform to this issue's requests. I will get to the SVGs now.

**Comment by danielhickman** [comment]
The gradient (not the object it's applied to) isn't pixel perfect.
```html
<linearGradient id="Outer-Circle-Gradient" gradientUnits="userSpaceOnUse" x1="105.5129" y1="5.4912" x2="86.4871" y2="186.5088">
	<stop offset="0" style="stop-color:#FAFAFA"/>
	<stop offset="1" style="stop-color:#BDBDBD"/>
</linearGradient>
```

**Comment by danielhickman** [comment]
Also the outer circle tint is 5% #202020 (not Material, I know this is all because of me, yadda yadda :wink:) and it also make it darker. Is the effect you were going for?

**Comment by crutchcorn** [comment]
Darn, wish you would have caught the circle gradient sooner but at least I haven't started working on the SVGs yet. However, the top tint IS intentional and I do not wish to change it. Nice catch though

**Comment by danielhickman** [comment]
So you're okay with using the not Material color? And wouldn't it be called a shade still? Or would you like to use "tint"?

**Comment by crutchcorn** [comment]
Yeah, I'm okay with the not Material color. I'd prefer to continue to use the word tint

**Comment by crutchcorn** [comment]
Just noticed something. @danielhickman Would it be easier to consistently use `<compound-path>` for glyphs? If it does, it makes generation of Tints and Shades easier as well. I might end up using it from here on out. However, if it does not make automation easier, I won't go back and edit previous icons

**Comment by danielhickman** [comment]
Converting a path (as opposed to multiple paths or a group) into a compound path shouldn't change anything at all, even for edges. I'm curious what you're experiencing. But, it shouldn't change anything; so go ahead :smile:.
